### PR TITLE
CODENVY-1271: fix agent launching in some cases

### PR DESCRIPTION
**Pull request by garagatyi**
### What does this PR do?
Fix AbstractAgentLauncher.
Code cleanup.

### What issues does this PR fix or reference?
codenvy/codenvy#1271

### CODE REVIEW

File: wsmaster/che-core-api-agent/src/test/resources/logback-test.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+-->
+<configuration>
+
+    <appender name="stdout" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%-41(%date[%.15thread]) %-45([%-5level] [%.30logger{30} %L]) - %msg%n</pattern>
+        </encoder>
+    </appender>
+    <root level="INFO">
```
Comment (sleshchenko) [inline wsmaster/che-core-api-agent/src/test/resources/logback-test.xml]: It would be better to set `ERROR` log level here

**Review by mshaposhnik — APPROVED** [review]

**Review by sleshchenko — APPROVED** [review]

**Review by akorneta — APPROVED** [review]

**Comment by garagatyi** [comment]
ci-build

File: wsmaster/che-core-api-agent/src/test/resources/logback-test.xml (hunk 1 above)
Comment (garagatyi) [reply wsmaster/che-core-api-agent/src/test/resources/logback-test.xml]: ok

**Review by skabashnyuk — APPROVED** [review]

**Comment by benoitf** [comment]
the branch needs to be rebased first due to history rewrite, thanks

**Pull request merged**